cfme/tests/candu/test_azone_graph.py

```diff
@@ -42,6 +42,8 @@ def azone(appliance, provider):
     blockers=[BZ(1671580, forced_streams=["5.9", "5.10"],
     unblock=lambda provider: not provider.one_of(AzureProvider))]
 )
+@pytest.mark.uncollectif(lambda provider, graph_type: provider.one_of(EC2Provider)
+    and graph_type == 'azone_memory' or graph_type == 'azone_disk')
```

Comment:
Optional: 
in such case; I would use `graph_type in ["azone_memory",  "azone_disk"]` .